pages/coinbase/homepage.py

```diff
@@ -1,45 +1,237 @@
+    cookie_dismiss_button_loc =     (by.XPATH, "//button[.='Dismiss']")
 
+    # NAVIGATION BAR
 
+    home_link_loc =                 (by.XPATH, "//a[@title='Home']")
+    prices_link_loc =               (by.XPATH, "//a[@title='Prices']")
+    products_menu_loc =             (by.XPATH, "//button[contains(text(),'Products')]")
+    company_menu_loc =              (by.XPATH, "//button[contains(text(),'Company')]")
+    earn_crypto_link_loc =          (by.XPATH, "//a[@title='Earn crypto']")
+    sign_in_link_loc =              (by.XPATH, "//a[@title='Sign in']")
+    header_get_started_button_loc = (by.XPATH, "//a[.='Get started']")
+
+         
+    # CREATE ACCOUNT
+
+    get_started_email_field_loc =   (by.XPATH, "//input[@type='email']")
+    get_started_button_loc =        (by.XPATH, "//input[@type='email']/following-sibling::button")
+
+    # CRYPTO CURRENCY MENU
+
+    bitcoin_text_link_loc =         (by.XPATH, "//h4[.='Bitcoin']")
+    bitcoin_BTC_link_loc =          (by.XPATH, "//h4[.='BTC']")
+    bitcoin_logo_link_loc =         (by.XPATH, "//img[@alt='Bitcoin logo']")
+    bitcoin_price_button_loc =      (by.XPATH, "//a[@href='/price/bitcoin']/button")
+
+    ethereum_text_link_loc =        (by.XPATH, "//h4[.='Ethereum']")
+    ethereum_ETH_link_loc =         (by.XPATH, "//h4[.='ETH']") 
+    ethereum_logo_link_loc =        (by.XPATH, "//img[@alt='Ethereum logo']")
+    ethereum_price_button_loc =     (by.XPATH, "//a[@href='/price/ethereum']/button")
+
+    bitcoinCash_text_link_loc =     (by.XPATH, "//h4[.='Bitcoin Cash']")        
+    bitcoinCash_BCH_link_loc =      (by.XPATH, "//h4[.='BCH']")
+    bitcoinCash_logo_link_loc =     (by.XPATH, "//img[@alt='Bitcoin Cash logo']")
+    bitcoinCash_price_button_loc =  (by.XPATH, "//a[@href='/price/bitcoin-cash']/button")
+
+    litecoin_text_link_loc =        (by.XPATH, "//h4[.='Litecoin']")
+    litecoin_LTC_link_loc =         (by.XPATH, "//h4[.='LTC']")
+    litecoin_logo_link_loc =        (by.XPATH, "//img[@alt='Litecoin logo']")
+    litecoin_price_button_loc =     (by.XPATH, "//a[@href='/price/litecoin']/button")
+
+    # EARNING LINKS BLCOK
+
+    start_earning_button_loc =       (by.XPATH, "//button[contains(@class,'EarnUpsellBanner')]")
+    start_earning_lower_button_loc = (by.XPATH, "//button[contains(@class,'EarnFooterBanner')]")
+    maker_link_loc =                 (by.XPATH, "//a[@href='/earn/maker']/div")
+    celo_link_loc =                  (by.XPATH, "//a[@href='/earn/celo']/div/div/h2")   #I don't love this xpath, but I feel it is the best way to locate this element
+    compound_link_loc =              (by.XPATH, "//a[@href='/earn/compound']/div/div/h2")   #ditto
+    EOS_link_loc =                   (by.XPATH, "//a[@href='/earn/eos']/div/div/h2")
+    view_more_link_loc =             (by.XPATH, "//a[@href='/earn']/div")
+
+    # MOBIL APPS
+    
+    android_app_link_loc =           (by.XPATH, "//a[contains(@href,'android')]")#='Android']")
```

Comment:
Can you please show this to me later? can't find it.

Reply:
![image](https://user-images.githubusercontent.com/28770835/96805353-11a73800-13d7-11eb-95d6-9cce994d7d88.png)

![image](https://user-images.githubusercontent.com/28770835/96805367-1f5cbd80-13d7-11eb-990d-af6bdf21b6eb.png)